### DISCUSSION TITLE
Haxe SDK install error

### DISCUSSION THREAD

**Discussion by hansagames** [colyseus/colyseus#573] [Q&A] [answered]
Hi,
tried to install haxe sdk from haxelib as described here https://docs.colyseus.io/getting-started/haxe-client/
but got follwing error
```
Error: Invalid filename : example/server/node_modules/tsx/dist/preflight..js
```

OS: OSX Monterey 12.6.5

Technically lib is installed but it doesnt show up as install failed.
Any ideas on this ?

Thank you

**Comment by endel** [comment]
Hi @hansagames, first time seeing this issue, when does it happen? What are the steps you took to see this error?

**Comment by hansagames** [comment]
Hi @endel 
I did `haxelib install colyseus` 
for context have Haxe 4.3.1 on macOS

**Reply by endel** [reply, marked as answer]
Thanks @hansagames, just fixed installation on `0.15.1`:

```
Downloading colyseus-0,15,1.zip...
Download complete: 0.24KB in 1.3s (0.1KB/s)
Download complete: 36.41KB in 1.2s (29.5KB/s)
Installing colyseus...
  Current version is now 0.15.1
Done
```

(Turns out the final haxelib uploaded had lots of example/unnecessary files https://github.com/colyseus/colyseus-haxe/commit/77a45e3b87b9475f5fa381ab06f1bb3615497f31)

Please feel free to give feedback on your experience using Colyseus!

**Reply by hansagames** [reply]
@endel thank you now works
